shell/packages/accounts-email-token/token_server.js

```diff
@@ -0,0 +1,163 @@
+var TOKEN_EXPIRATION_MS = 15 * 60 * 1000;
+
+var cleanupExpiredTokens = function() {
+  Meteor.users.update({}, {
+    $pull: {
+      "services.emailToken.tokens": {
+        createdAt: {$lt: new Date(Date.now() - TOKEN_EXPIRATION_MS)}
+      }
+    }
+  }, { multi: true });
+};
+
+Meteor.startup(cleanupExpiredTokens);
+// Tokens can actually last up to 2 * TOKEN_EXPIRATION_MS
+Meteor.setInterval(cleanupExpiredTokens, TOKEN_EXPIRATION_MS);
+
+Accounts._checkToken = function (user, token) {
```

Comment:
It doesn't appear than anyone actually calls this as `Accounts._checkToken`, so why put it in `Accounts`?